bciers/apps/administration/app/components/profile/ProfileForm.tsx

```diff
@@ -72,8 +70,8 @@ export default function ProfileForm({ formData, isCreate }: Props) {
       setIsSuccess(false);
     }, 3000);
     if (isCreate) {
-      // 🛸 Routing: after the update is complete, navigate to the dashboard
-      router.push("/");
+      // 🛸 Redirect: after the update is complete, navigate to the dashboard
+      window.location.href = "./";
```

Comment:
Teachable moment for Brianna--why is this preferable to useRouter?

Reply:
@BCerki 

1. `router.push("/");` needed to be changed from routing to administration to routing to dashboard.
2. Changing to  `router.push("../");` reflected `http://localhost:3000/dashboard` in the browser but reflected ` /_next/static/chunks/app/dashboard/page.js` in the app and cause a 404 error:
![image](https://github.com/user-attachments/assets/5ed338ce-b4dc-4880-a49e-66b849c5bbc0)

So, I used `window.location.href = "./";` to reload the app and let the middlewares and rewrites "start fresh".

Perhaps, this could also have been solved within `bciers/apps/dashboard/next.config.js` using `beforeFiles` config